modules/ossm-sidecar-validate-kiali.adoc

```diff
@@ -0,0 +1,23 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc
+////
+:_content-type: CONCEPT
+[id="ossm-validating-sidecar_{context}"]
+= Validating sidecar injection
+
+The Kiali console offers several ways to validate whether or not your applications, services, and workloads have a sidecar proxy.
+
+The *Graph* page displays a node badge indicating a *Missing Sidecar* on the following graphs:
+
+* App graph
+* Versioned app graph
+* Workload graph
+
+The *Applications* page displays a *Missing Sidecar* icon in the *Details* column for any applications in a namespace that do not have a sidecar.
+
+The *Workloads* page displays a *Missing Sidecar* icon in the *Details* column for any applications in a namespace that do not have a sidecar.
+
+The *Services* page displays a *Missing Sidecar* icon in the *Details* column for any applications in a namespace that do not have a sidecar. Note that when there are multiple versions of a service, you use the *Service Details* page to view *Missing Sidecar* icons.
+
+The *Workload Details* page has a special unified *Logs* tab that lets you view and correlate application and proxy logs. You can view the Envoy logs as another way to validate sidecar injection for your application workloads.
```

Comment:
Worth mentioning that the "Workload Details" offer an "Envoy" tab where user can navigate in the details of the proxy:
https://kiali.io/docs/features/details/#envoy